src/openrct2-ui/WindowManager.cpp

```diff
@@ -342,7 +342,6 @@ class WindowManager final : public IWindowManager
                 {
                     window_close_construction_windows();
                     _currentRideIndex = static_cast<ride_id_t>(rideIndex);
-                    w = OpenWindow(WC_RIDE_CONSTRUCTION);
```

Comment:
Are you sure about this change. You are now no longer opening the construction window if it isn't already opened.

Reply:
Nope, that slipped in. Only the assignment to `w` is useless.